source/cli.coffee

```diff
@@ -33,14 +34,13 @@ if process.argv.includes "--help"
       --version        Show the version number
       -o / --output XX Specify output directory and/or extension, or filename
       -c / --compile   Compile input files to TypeScript (or JavaScript)
+      --config XX      Specify a config file (default finds the nearest config.civet, .civetconfig or civet.json file)
       --js             Strip out all type annotations; default to .jsx extension
       --ast            Print the AST instead of the compiled code
       --inline-map     Generate a sourcemap
       --no-cache       Disable compiler caching (slow, for debugging)
 
     You can use - to read from stdin or (prefixed by -o) write to stdout.
-
```

Comment:
These newlines should stay in, they are necessary to insert a newline after printing out the help message.